src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-700.mdx

```diff
@@ -0,0 +1,26 @@
+---
+subject: Android agent
+releaseDate: '2023-06-08'
+version: 7.0.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.0.0.zip'
+---
+
+
+## New
+
```

Comment:
Add block to announce and describe major version increase: include new support levels, deprecations, caveats, what is no longer supported. Check version 6.0.0 for example

Reply:
Consult with doc team